specification/security/resource-manager/Microsoft.Security/preview/2024-07-01-preview/securityConnectors.json

```diff
@@ -0,0 +1,1707 @@
+      "type": "object",
+      "description": "The CSPM monitoring for AWS offering",
+      "x-ms-discriminator-value": "CspmMonitorAws",
+      "allOf": [
+        {
+          "$ref": "#/definitions/cloudOffering"
+        }
+      ],
+      "properties": {
+        "nativeCloudConnection": {
+          "type": "object",
+          "description": "The native cloud connection configuration",
+          "properties": {
+            "cloudRoleArn": {
+              "type": "string",
+              "description": "The cloud role ARN in AWS for this feature"
+            }
+          }
+        }
+      }
+    },
+    "defenderForContainersAwsOffering": {
+      "type": "object",
+      "description": "The Defender for Containers AWS offering",
+      "x-ms-discriminator-value": "DefenderForContainersAws",
+      "allOf": [
+        {
+          "$ref": "#/definitions/cloudOffering"
+        }
+      ],
+      "properties": {
+        "kubernetesService": {
+          "type": "object",
+          "description": "The kubernetes service connection configuration",
+          "properties": {
+            "cloudRoleArn": {
+              "type": "string",
+              "description": "The cloud role ARN in AWS for this feature used for provisioning resources"
+            }
+          }
+        },
+        "kubernetesDataCollection": {
+          "type": "object",
+          "description": "The kubernetes data collection connection configuration",
+          "properties": {
+            "cloudRoleArn": {
+              "type": "string",
+              "description": "The cloud role ARN in AWS for this feature used for reading data"
+            }
+          }
+        },
+        "cloudWatchToKinesis": {
+          "type": "object",
+          "description": "The cloudwatch to kinesis connection configuration",
+          "properties": {
+            "cloudRoleArn": {
+              "type": "string",
+              "description": "The cloud role ARN in AWS used by CloudWatch to transfer data into Kinesis"
+            }
+          }
+        },
+        "kinesisToS3": {
+          "type": "object",
+          "description": "The kinesis to s3 connection configuration",
+          "properties": {
+            "cloudRoleArn": {
+              "type": "string",
+              "description": "The cloud role ARN in AWS used by Kinesis to transfer data into S3"
+            }
+          }
+        },
+        "enableAuditLogsAutoProvisioning": {
+          "type": "boolean",
+          "description": "Is audit logs data collection enabled"
+        },
+        "enableDefenderAgentAutoProvisioning": {
+          "type": "boolean",
+          "description": "Is Microsoft Defender for Cloud Kubernetes agent auto provisioning enabled"
+        },
+        "enablePolicyAgentAutoProvisioning": {
+          "type": "boolean",
+          "description": "Is Policy Kubernetes agent auto provisioning enabled"
+        },
+        "kubeAuditRetentionTime": {
```

Comment:
kubeAuditRetentionTimeInHours ?

Reply:
This version is build on top of the older version with the goal of supporting a new environment only, without the intention to make further unrelated changes